storage/include/storage/storage_metrics.h

```diff
@@ -3,7 +3,7 @@
 #pragma once
 
 #include "Metrics.hpp"
-
+#include <mutex>
```

Comment:
I think we don't need that include anymore.

Reply:
will be removed